megatron/megatron/core/pipeline_parallel/p2p_communication.py

```diff
@@ -127,35 +129,36 @@ def _batched_p2p_ops(
     group: torch.distributed.ProcessGroup
 ):
     ops = []
+
     if tensor_send_prev is not None:
         send_prev_op = torch.distributed.P2POp(
             torch.distributed.isend,
             tensor_send_prev,
-            get_pipeline_model_parallel_prev_rank(),
+            get_pipeline_model_parallel_prev_rank(group=group),
```

Comment:
Ensure that the same functions of _p2p_ops are explicitly assigned to the group.

Reply:
OK.